main.nf

```diff
@@ -912,28 +1038,34 @@ workflow.onComplete {
     email_fields['summary']['Date Completed'] = workflow.complete
     email_fields['summary']['Pipeline script file path'] = workflow.scriptFile
     email_fields['summary']['Pipeline script hash ID'] = workflow.scriptId
-    email_fields['summary']['Container'] = workflow.container
-    if( workflow.repository ) email_fields['summary']['Pipeline repository Git URL'] = workflow.repository
-    if( workflow.commitId ) email_fields['summary']['Pipeline repository Git Commit'] = workflow.commitId
-    if( workflow.revision ) email_fields['summary']['Pipeline Git branch/tag'] = workflow.revision
+    if (workflow.repository) email_fields['summary']['Pipeline repository Git URL'] = workflow.repository
+    if (workflow.commitId) email_fields['summary']['Pipeline repository Git Commit'] = workflow.commitId
+    if (workflow.revision) email_fields['summary']['Pipeline Git branch/tag'] = workflow.revision
+    if (workflow.container) email_fields['summary']['Docker image'] = workflow.container
```

Comment:
Is this the url to the container, or could this also be the path to the singularity image?

Reply:
Either, and it's also a duplicate log. I already removed it from the template the other day: https://github.com/nf-core/tools/pull/452

Reply:
Then if it's either, it should not be `Docker image` but `Container` or something like that